buildconfig/Jenkins/Conda/run-tests

```diff
@@ -92,10 +92,16 @@ else
     SYSTEM_TEST_EXECUTABLE=$WORKSPACE/build/systemtest
 fi
 
+
+
 # Run unit tests
 cd $WORKSPACE/build
+# ctest log
+# CTEST_LOG_FILENAME="$OSTYPE.log"
+##mkdir for ctest log
```

Comment:
Remove commented out code

Reply:
resolved in [eafaa4e](https://github.com/mantidproject/mantid/commit/eafaa4ec5b4cf18362e64b915e63a55ac702f1af)